pkg/asset/templates/content/bootkube/pull.go

```diff
@@ -9,12 +9,12 @@ import (
 )
 
 const (
-	pullFileName = "pull.yaml.template"
+	pullFileName = "pull.json.template"
```

Comment:
This seems unrelated?

Reply:
Ah, I see from the commit message:

> fixes the template for pull as the contents were json but file was names yaml

But JSON is a subset of YAML, so `.yaml` was accurate, but less specific.  Anyhow, I'm fine with this rename if you don't want to bother splitting it out ;).

Reply:
I have mentioned that in https://github.com/openshift/installer/pull/1363/commits/c52894b2c2064e5cb8a6d736ccdbe29279d18ceb
```
modifies the manifest target to use the filename from template assets, rather than duplicating the filename.

fixes the template for pull as the contents were json but file was names yaml
```

Reply:
so mco requires the file name to be [`pull.json`](https://github.com/openshift/installer/blob/ade95dd3cbb88efc6afe601dadce8e7977bc3053/data/data/bootstrap/files/usr/local/bin/bootkube.sh.template#L178)

and when I was making the change it got confused wriring the write file name to write asset.... :( so I made the change  https://github.com/openshift/installer/commit/c52894b2c2064e5cb8a6d736ccdbe29279d18ceb

Reply:
+1 for consistency between the name of the template and the name of the file generated.